po/de.po

```diff
@@ -323,14 +323,15 @@ msgstr "Besuche Piper auf GitHub"
 #: data/ui/AboutDialog.ui.in:13
 msgid "translator-credits"
 msgstr "Peter Hutterer <peter.hutterer@who-t.net>, 2017-2019"
+"\nNicolai Weitkemper <kontakt@nicolaiweitkemper.de>, 2020"
```

Comment:
I'm not sure if it's appropriate to add myself to the list of contributors for these small corrections. I also have no idea whether a newline is right here.

Reply:
mostly depends on whether you want the claim to fame :) this is imo a leftover from early approaches where all this was only tracked in the source files and not in other trackers like github.

The newline should be there iirc, just check the About dialog for correctness. But isn't this bit autogenerated anyway?

Reply:
The newline looks fine. These lines weren't auto-generated for me.